cms/templates/partials/catalog-topics-section.html

```diff
@@ -0,0 +1,17 @@
+{% load wagtailcore_tags wagtailembeds_tags %}
+
+<div class="catalog-topics-section">
+  <div class="container">
+    <h1>DISCOVER THE xPRO CATALOG</h1>
```

Comment:
I think this might be something, to be set through wagtail, Maybe we can introduce a text field in the HomePage model. I assume this because the titles of the other sections are configurable through CMS. 

But I might be taking it too far, I suspect this title has less tendency to change. So this is just a thought, You can leave it.